tools/webpack.config.js

```diff
@@ -228,6 +228,8 @@ const config = {
 // Optimize the bundle in release (production) mode
 if (!isDebug) {
   config.plugins.push(new webpack.optimize.UglifyJsPlugin({
+    parallel: true,
+    cache: true,
```

Comment:
What do these two?

Reply:
Execute in paralell and cache some stuff for a speedup :P